packages/common/src/codegen/utils/loadUserTypesFile.ts

```diff
@@ -31,8 +31,9 @@ function loadUserTypesFile(
   data: string;
 } {
   if (unresolvedFilePath.at(0) === ".") {
+    const relativePath = path.relative(outputBaseDirectory, unresolvedFilePath);
     return {
-      filePath: path.relative(outputBaseDirectory, unresolvedFilePath),
+      filePath: "./" + relativePath, // solc doesn't like relative paths without "./"
```

Comment:
is this gonna muck with windows at all?

Reply:
no, it's just relevant for imports in solidity files, which are processed by `solc`. We do the same for other relative imports in table libraries already.